configure.ac

```diff
@@ -698,6 +698,17 @@ if test -d ${srcdir}/libphobos; then
     fi
 fi
 
+# Disable libgrust when not using the rust language
+if test -d ${srcdir}/libgrust; then
```

Comment:
This `if` isn't actually necessary here, as you're not attempting to use files from `${srcdir}/libgrust/`.  (Background is that the top-level build system is shared between GCC and Binutils/GDB, and `${srcdir}/libgrust/` doesn't exist for the latter, for example.)  However, as such usage is going to appear at some point in time (like "Disable libphobos on unsupported systems" just above, for example), it's OK to leave in that `if`.

Reply:
TBH i wasn't sure why it was put everywhere then I read that some binutils build share that script and thought I better put a seamlessly useless condition rather than breaking an obscure bintutils build somewhere. I'm not yet sure if we want to disable libgrust at some point though.

Reply:
> I'm not yet sure if we want to disable libgrust at some point though. Or is this only disabled for binutils/GDB ?

It'll have to be disabled for all configurations where it doesn't build (or, rather: only be enabled for configurations where it is supported/known to build).  See <https://github.com/Rust-GCC/gccrs/blob/master/libsanitizer/configure.tgt> as a random example.  Similar to the libsanitizer case, I suppose that for libgrust we'll also have that decided that for each "sub-library" individually: the generic Procedural Macros support library is always built (for it doesn't contain any configuration specifics, supposedly), but, for example, the Rust Standard Library (imported from upstream Rust) only for configurations where it's known to build.